src/Job/NormalJobOptions.ts

```diff
@@ -0,0 +1,18 @@
+import { Job, JobEnv } from './Job'
+
+/**
+ * An interface describing options for a {@link NormalJob}.
+ *
+ * target: The target file that the Job will produce. Must be unique between jobs in the same dependency graph.
+ * commands: An array of the commands to run to build the target.
+ * prerequisiteJobs: A set containing all of this job's prerequisite Jobs.
+ * prerequisiteFiles: A set containing all of this job's prerequisite files
```

Comment:
In what sense are we using "prerequisite job" and "prerequisite file" here? How are the 2 distinct?  Can we give a brief example to make the documentation a bit more clear? 

```suggestion
 * prerequisiteFiles: A set containing the names of all files needed by the job to run (eg. "sourceCodeToCompile.js"). 
```